scripts/deblur

```diff
@@ -198,11 +198,13 @@ def dereplicate(seqs_fp, output_fp, min_size, log_level, log_file):
                    "database. For multiple databases, the order "
                    "must follow that of --ref-fp, for example, "
                    "--ref-db-fp db1.idx --ref-db-fp db2.idx ..")
-@click.option('--negate', '-n', required=False, default=False, is_flag=True,
-              show_default=True, type=bool,
-              help="Discard all sequences aligning to the database "
-                   "passed via --ref-fp option")
-@click.option('--threads-per-sample', '-t', required=False, type=int,
+@click.option('--negate/--only-pos', '-n/', required=False, default=True, is_flag=True,
```

Comment:
i think it's a good idea :)
now that positive runs by default after creating the biom table, should be enough.
(there is an edge case where you are only interested in 16s then running positive mode in the beginning - before deblur - will be more efficient compared to running negative mode, deblurring and then positive filtering)
but i think maybe simplicity is more important than this edge case?
so let's do it !